Mathlib/Algebra/Lie/Killing.lean

```diff
@@ -26,12 +27,12 @@ We define the trace / Killing form in this file and prove some basic properties.
  * `LieModule.traceForm_eq_zero_of_isNilpotent`
  * `killingForm`
  * `LieAlgebra.IsKilling`
+ * `LieAlgebra.IsKilling.ker_restrictBilinear_of_isCartanSubalgebra_eq_bot`
  * `LieAlgebra.IsKilling.isSemisimple`
```

Comment:
could you expand these, by explaining quickly what each definition or theorem is saying?

Reply:
Good point, done!